ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.13.yaml

```diff
@@ -189,13 +189,12 @@ tests:
   steps:
     cluster_profile: aws-qe
     env:
-      ADDITIONAL_ENABLED_CAPABILITIES: CSISnapshot
```

Comment:
If we keep the setting, would that affect the following testing?

Reply:
No, we have discussed it in thread https://github.com/openshift/release/pull/43227#discussion_r1339470158. So let's updated it to comply with the case correctly to avoid misunderstanding here.

Reply:
Do you mind to update the same job in openshift-openshift-tests-private-release-4.15__amd64-nightly-4.15-upgrade-from-stable-4.14.yaml

Reply:
sure, let me updated all other related jobs

Reply:
Ah, sounds we also need to update the ci job name to remove `addcap` keyword together.

If possible, can you also help update the fresh install e2e jobs ones togher?

```
[jianlinliu@jialiu-laptop openshift-tests-private]$ grep -r "as: aws.*sts" *release-4.1{4,5}*|grep base
openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.13.yaml:- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28
openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28
openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28-destructive
openshift-openshift-tests-private-release-4.14__arm64-nightly-4.14-upgrade-from-stable-4.13.yaml:- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28
openshift-openshift-tests-private-release-4.14__arm64-nightly.yaml:- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28
openshift-openshift-tests-private-release-4.14__arm64-nightly.yaml:- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28-destructive
openshift-openshift-tests-private-release-4.15__amd64-nightly-4.15-upgrade-from-stable-4.14.yaml:- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28
openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml:- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28
openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml:- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28-destructive
openshift-openshift-tests-private-release-4.15__arm64-nightly-4.15-upgrade-from-stable-4.14.yaml:- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28
openshift-openshift-tests-private-release-4.15__arm64-nightly.yaml:- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28
openshift-openshift-tests-private-release-4.15__arm64-nightly.yaml:- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28-destructive
```

Reply:
@jianlinliu Are you sure we need update fresh install jobs too? I see there are ADDITIONAL_ENABLED_CAPABILITIES for them.
```
- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28
  cron: 24 21 27 * *
  steps: 
    cluster_profile: aws-qe
    env:
      ADDITIONAL_ENABLED_CAPABILITIES: CSISnapshot MachineAPI
      BASE_DOMAIN: qe.devcluster.openshift.com
      BASELINE_CAPABILITY_SET: None
```

Reply:
Yeah, the jobs were added on purpose to cover the test case, on that time, I thought the additional cap is a requirement for the case, so I added  `CSISnapshot` coverage for both fresh install and upgrade. For `MachineAPI`, there is a known issue when running ipi install + none basecap, so enable `MachineAPI` as a workaround, but that is not a test point.

Reply:
ack, then let me update install job accordingly